src/main/java/raythx98/duke/task/ToDos.java

```diff
@@ -0,0 +1,78 @@
+package raythx98.duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoField;
+
+import raythx98.duke.exception.DukeException;
+
+/**
+ * Represents a specific task which is a to do.
+ */
+public class ToDos extends Task {
+
+    protected LocalDate date = null;
+    protected String by = null;
+
+    public ToDos(String description, String by) {
+        super(description);
+        tag = "T";
+        String[] bySplit = by.split(" ", 2);
+        date = LocalDate.parse(bySplit[0]);
+        if (bySplit.length > 1) {
+            this.by = bySplit[1];
+        }
+    }
+
+    public ToDos(String description) {
+        super(description);
+        tag = "T";
+    }
+
+    public ToDos(String ... taskDescriptions) throws DukeException {
+        super(taskDescriptions[2]);
+        tag = "T";
+        if (taskDescriptions.length == 3) {
+        } else if (taskDescriptions.length == 5) {
+            this.date = LocalDate.parse(taskDescriptions[3]);
+            this.by = taskDescriptions[4];
+        } else {
+            throw new DukeException("Task loading error...");
+        }
+        if (taskDescriptions[1].equals("1")) {
+            this.markAsDone();
+        }
+    }
+
+    @Override
+    public String getTaskType() {
+        return tag;
+    }
+
+    public String toPrint() {
```

Comment:
method name should be a commanding word starting with a verb, such as 'getStringForPrinting'